trustregistry/crud.py

```diff
@@ -208,6 +236,9 @@ def delete_schema(db_session: Session, schema_id: str) -> db.Schema:
 class ActorAlreadyExistsException(Exception):
     """Raised when attempting to create an actor that already exists in the database."""
 
+    def __init__(self, message):
+        self.message = message
```

Comment:
Optional: With this custom def __init__, you can also override the string method:
```
    def __str__(self):
        return self.message
```

This is allows you to use `e` in a f-string context, without needing to say strictly call e.message